proposals/NNNN-remove-customization-points.md

```diff
@@ -0,0 +1,116 @@
+# Remove Some Customization Points from the Standard Library's `Collection` Hierarchy
+
+* Proposal: [SE-NNNN](NNNN-remove-customization-points.md)
+* Author: [Ben Cohen](https://github.com/airspeedswift)
+* Review Manager: TBD
+* Status: **Awaiting implementation**
+* Implementation: [apple/swift#19769](https://github.com/apple/swift/pull/19769)
+
+## Introduction
+
+This proposal removes four customization points from protocols in the
+standard library:
+
+- `map` and `forEach` from `Sequence`
+- `first` from `Collection`
+- `last` on `BidirectionalCollection`
+
```

Comment:
Could this be a little more clear up front that there isn't really a source-compatibility impact? Something like:

> The default implementations of these symbols will remain, so sequences and collections will continue to have the same operations available that they do today.

Reply:
Good point. Added.